src/main/resources/changesets/changelog-consent-14.0.xml

```diff
@@ -28,7 +28,7 @@
     <changeSet author="vvicario" id="58">
         <validCheckSum>ANY</validCheckSum>
         <addForeignKeyConstraint baseColumnNames="consent" baseTableName="matchentity"
-                                 constraintName="fkMatchConsent" deferrable="false" initiallyDeferred="false"
+                                 constraintName="fkmatchconsent" deferrable="false" initiallyDeferred="false"
```

Comment:
is changing the old changeset intentional? i'm not sure how these work - would this be updated in prod, too?

Reply:
Ah, good eye. Yes, this is intentional. Originally, in mysql, this was camel cased. When we migrated to postgres, this constraint was all lower cased. The changelog didn't need to be updated since the file had already been run and didn't need to be re-run. However, when we run tests, all changelog files ARE re-run and we hit a naming error because `fkMatchConsent` doesn't match `fkmatchconsent`. The current constraint name is actually lower cased, so this change makes it consistent with what's real.